modules/platforms/build.gradle

```diff
@@ -36,6 +42,9 @@ configurations {
     platformsHeaders
 }
 
+def manylinuxWheelsContainerName = "ignite_manylinux_wheels"
+def manylinuxWheelsImageName = "ignite_manylinux_python_wheels_build"
```

Comment:
where is `ignite_manylinux_python_wheels_build` built?

Reply:
In `createManyLinuxWheelsContainer` task